xsd/netex_framework/netex_genericFramework/netex_contact_support.xsd

```diff
@@ -97,9 +97,9 @@ Rail transport, Roads and Road transport
 		<xsd:restriction base="xsd:NMTOKEN">
 			<xsd:enumeration value="any"/>
 			<xsd:enumeration value="information"/>
-			<xsd:enumeration value="reservations"/>
+			<xsd:enumeration value="reservation"/>
```

Comment:
Where do the plurals come from?

Reply:
They were in the standard. But all to other enum values are not plurals, so i harmonized them.